src/collaborative_platform/apps/projects/assets/src/components/App.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+import taxonomy from './def_taxonomy.js';
+import ProjectDescriptionSection from './ProjectDescriptionSection.js';
+import TaxonomySection from './TaxonomySection.js';
+
+class App extends React.Component {
+  constructor(props){
+    super(props);
+    
+    this.state = {
+      title: '',
+      description: '',
+      certScheme: Object.entries(taxonomy.taxonomy),
+    };
+
+    this.createProject = this.createProject.bind(this);
+    this.updateCertScheme = this.updateCertScheme.bind(this);
+  }
+
+  componentDidMount(){
+    document.getElementById('create-project-button')
+        .addEventListener('click', this.createProject);
+      
+  }
+
+  updateCertScheme(scheme){
+    this.setState({certScheme: scheme});
+  }
+
+  createProject(e){
+    const data = {
+      title: this.state.title,
+      description: this.state.description,
+      categories: this.state.certScheme,
+      "taxonomy.xml_id_1": this.state.certScheme[0],
+      "taxonomy.xml_color_1": this.state.certScheme[1].color,
+      "taxonomy.xml_desc_1": this.state.certScheme[1].desc,
+      "taxonomy.xml_id_2": this.state.certScheme[0],
+      "taxonomy.xml_color_2": this.state.certScheme[1].color,
+      "taxonomy.xml_desc_2": this.state.certScheme[1].desc,
+      "taxonomy.xml_id_3": this.state.certScheme[0],
+      "taxonomy.xml_color_3": this.state.certScheme[1].color,
+      "taxonomy.xml_desc_3": this.state.certScheme[1].desc,
+      "taxonomy.xml_id_4": this.state.certScheme[0],
+      "taxonomy.xml_color_4": this.state.certScheme[1].color,
+      "taxonomy.xml_desc_4": this.state.certScheme[1].desc,
+    };
```

Comment:
Hi @Janchorizo.
For this to work it should look like below:
`  createProject(e){
    const data = {
      title: this.state.title,
      description: this.state.description,
      categories: this.state.certScheme,
      "taxonomy.xml_id_1": this.state.certScheme[0][0],
      "taxonomy.color_1": this.state.certScheme[0][1].color,
      "taxonomy.desc_1": this.state.certScheme[0][1].desc,
      "taxonomy.xml_id_2": this.state.certScheme[1][0],
      "taxonomy.color_2": this.state.certScheme[1][1].color,
      "taxonomy.desc_2": this.state.certScheme[1][1].desc,
      "taxonomy.xml_id_3": this.state.certScheme[2][0],
      "taxonomy.color_3": this.state.certScheme[2][1].color,
      "taxonomy.desc_3": this.state.certScheme[2][1].desc,
      "taxonomy.xml_id_4": this.state.certScheme[3][0],
      "taxonomy.color_4": this.state.certScheme[3][1].color,
      "taxonomy.desc_4": this.state.certScheme[3][1].desc,
    };`

But since it's not the best format nevertheless, and in React it's easy to change,  I'll change it to dicts, and commit to develop.